elyra/api/elyra.yaml

```diff
@@ -342,6 +342,35 @@ paths:
                     type: string
                     description: The pipeline export path.
 
+  /elyra/contents/properties/{path}:
+    get:
+        tags:
+        - properties
+        summary: Get a dictionary of properties for a file.
+        parameters:
+        - name: path
+          in: path
+          description: The path to the file to be parsed, relative to the JupyterLab server root directory.
+          required: true
+          schema:
+            type: string
+        responses:
+          200:
+            description: The dictionary of properties found while parsing the file.
+            content:
+              application/json:
+                schema:
+                  type: object
+                  description: A dictionary of property dictionaries. Each property dictionary consists of a key for each instance found and its value (if applicable), or, if parsing fails, is empty.
+                  oneOf:
+                    - $ref: '#/components/schemas/ResourceBody'
+                    - $ref: '#/components/schemas/ResourceBodyEmpty'
```

Comment:
Since the empty response is literally an empty response (and normal), I don't think we need an explicit entity for that.